src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="wclassth=device-wclassth, initial-scale=1.0"
```

Comment:
There is a typo in the viewport meta tag: 'wclassth' should be 'width'. This will prevent the viewport from being set correctly on mobile devices.